data/json/items/book/maps.json

```diff
@@ -48,7 +48,17 @@
     "use_action": {
       "type": "reveal_map",
       "radius": 180,
-      "terrain": [ "hiway", "road", "bridge", "s_pharm", "s_gun", "s_grocery", "s_hunting", "mil_surplus", "s_gas" ],
+      "terrain": [
+        "hiway",
+        "road",
+        "bridge",
+        "s_pharm",
+        "s_gun",
+        "s_grocery",
+        "s_hunting",
+        "mil_surplus",
+        "s_gas"
+      ],
```

Comment:
The convention is to put an array or an object in a single line if it's short enough. The JSON formatter will complain if you wrap it this way.

Reply:
i see. i'll change that one back. how short is short enough?

Reply:
I think it's 120 characters, but you can just use the [json linting tool](http://dev.narc.ro/cataclysm/format.html). Copy a changed file into it, hit "lint", and copy the output back.

Reply:
Paste the json in the web linter and it will format it for you. http://dev.narc.ro/cataclysm/format.html